runtimes/manta/src/chain_spec.rs

```diff
@@ -417,6 +345,10 @@ pub fn manta_testnet_config_genesis(
 		grandpa: GrandpaConfig {
 			authorities: vec![],
 		},
+		manta_pay: MantaPayConfig {
+			owner: hex!["12b73670c56f4fcd319636bdd6ec4a803ae2d06fdbc715087524a5151395d16c"].into(),
```

Comment:
Why is this not `get_account_id_from_seed::<sr25519::Public>("Alice")` like in the `testnet_genesis` function? Is it someone else?

Reply:
Yeah, for `dev/local` we use Dev account, for Dolphin we use a specific account.